cmd/metal-api/internal/service/switch-service.go

```diff
@@ -191,13 +194,22 @@ func (r *switchResource) findSwitches(request *restful.Request, response *restfu
 
 func (r *switchResource) deleteSwitch(request *restful.Request, response *restful.Response) {
 	id := request.PathParameter("id")
+	force, err := strconv.ParseBool(request.PathParameter("force"))
+	if err != nil {
+		r.sendError(request, response, httperrors.BadRequest(err))
```

Comment:
Missing return. Code will continue after this.

Reply:
Shit, sorry

Reply:
@iljarotar please create a FIX PR